ftplugin/ledger.vim

```diff
@@ -328,3 +348,11 @@ endf "}}}
 function! s:count_expression(text, expression) "{{{2
   return len(split(a:text, a:expression, 1))-1
 endf "}}}
+
+" Move the cursor to the specified column, filling the line with spaces if necessary.
+function! s:goto_col(pos) "{{{2
```

Comment:
If I remember correctly `s:` refers to a function defined local to the current script (file). Thus this definition needs to be moved to `autoload/ledger.vim`.